### ISSUE TITLE
Movie cells appear for one tab, but not the other

### ISSUE THREAD

**Issue by liuwinshen** [uchicago-mobi/MPCS51030-2019-Winter-Forum#37]
I have the same IB outlets from my storyboard to MovieCell, nearly identical code for my two tabs, and the corresponding reuse identifier, but the query results only show up for one tab. 

In debugging, the iTunes query succeeds for both tabs.

What are other things I can check to debug this?

**Comment by jarofmy** [comment]
It sounds like the second tab may not be referencing the outlets within the cell for some reason. I'm not sure if you happened to set up separate IBOutlets for Chicago vs. New York views (i.e. chicagoMovieLabel and newyorkMovieLabel). But I found that one IBOutlet object in MovieCell is able to reference/connect to multiple items in both TableViewControllers. For example, in your MovieTableViewCell, if you have `@IBOutlet weak var movieLabel`, it could connect to the movieLabel in both the TableViewControllers.  This sure ensure that both tabs are referencing the same IBOutlet and should act the same way.

If all else fails, I found that breaking the connections from an object on the storyboard, and re-connecting it usually helps as well.
